pkg/operator/ceph/disruption/clusterdisruption/reconcile.go

```diff
@@ -262,7 +263,17 @@ func (r *ReconcileClusterDisruption) deleteDrainCanaryPods(namespace string) err
 }
 
 func (r *ReconcileClusterDisruption) deleteLegacyPDBForOSD(namespace string) error {
```

Comment:
@sp98 When did we get rid of the legacy PDBs? I think we can get rid of this method now since it's already been at least a release since we removed the legacy PDBs.

Reply:
We added this in rook 1.5 in [this](https://github.com/rook/rook/pull/6497) PR.  So should be safe to remove now.

Reply:
@sp98  We want to remove it in this PR or maybe in a separate PR?

Reply:
separate PR, IMO.